src/components/templates/ArtPiece/ArtPiece.scss

```diff
@@ -17,7 +17,7 @@
   .video-chat-wrapper {
     display: grid;
     margin: 1rem 0;
-    grid-template-columns: repeat(6, 1fr);
+    grid-template-columns: repeat(9, 1fr);
```

Comment:
Just so I can understand it better, can you explain what this is doing/how it works please?

Reply:
I just spoke with Ed, and this should be 5 across. 1 video takes 3 columns. Not sure why, but we are in a hurry for the event and I don't have the time to look into it.

Reply:
No worries, don't need to clean it up now. Was just curious/for my own knowledge.